Lib/site.py

```diff
@@ -498,40 +498,55 @@ def register_readline():
         PYTHON_BASIC_REPL = False
 
     import atexit
+
+    try:
+        try:
+            import readline
+        except ImportError:
+            readline = None
+        else:
+            import rlcompleter  # noqa: F401
```

Comment:
Can this import ever fail?

Reply:
I don't know. In case of doubt, I prefer to keep the old behavior: catch and ignore ImportError.